code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -97,6 +97,17 @@
 				if(T.canBuildPlating(S) == BUILD_SUCCESS)
 					qdel(L)
 
+/obj/item/stack/tile/wood/attackby(obj/item/weapon/W as obj, mob/user as mob)
```

Comment:
as syntax is not used outside of verbs and input()s, so you should leave it out in the future.